tests/Agent/IntegrationTests/ContainerApplications/AwsSdkTestApp/Controllers/AwsSdkDynamoDBController.cs

```diff
@@ -0,0 +1,122 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using AwsSdkTestApp.AwsSdkExercisers;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace AwsSdkTestApp.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class AwsSdkDynamoDBController : ControllerBase
+    {
+        private readonly ILogger<AwsSdkDynamoDBController> _logger;
+
+        public AwsSdkDynamoDBController(ILogger<AwsSdkDynamoDBController> logger)
+        {
+            _logger = logger;
+
+            _logger.LogInformation("Created AwsSdkDynamoDBController");
+        }
+
+        // GET: /AwsSdkDynamoDB/CreateTable?tableName=tableName
```

Comment:
nit: comment doesn't match actual route name. If you're not going to do both sync and async methods in the exerciser, you probably don't need to name the routes with `Async` suffixes.

Reply:
I fixed the comment rather than changing the name of the routes.  I thought I saw a compiler warning earlier during development that was unhappy when a controller method that is async isn't named something with `Async` in the name, although I just tried to reproduce that and couldn't.